pkgs/tools/security/vault/default.nix

```diff
@@ -7,27 +7,32 @@ let
     rev = "e2f59b64be1fa5430fa05c91b6274284de4ea77c";
     sha256 = "10m75rp3hy71wlmnd88grmpjhqy0pwb9m8wm19l0f463xla54frd";
   };
-in buildGoPackage rec {
+in stdenv.mkDerivation rec {
```

Comment:
Not sure if I missed the comment, but why the switch away from `buildGoPackage`? I'm not saying you shouldn't, but I'd want to make sure any annoyances you run into with `buildGoPackage` get back to @kamilchm and get improved over time.